requirements.txt

```diff
@@ -17,6 +17,7 @@ pyyaml
 # pytorch build script pins numpy version
 # https://github.com/pytorch/builder/blob/ae5c82e65cb3d8bac6df50e742a195019af91ad3/wheel/build_wheel.sh#L145
 numpy==1.21.2
+opencv-python==4.7.0.72
```

Comment:
Can we add link to the upstream bug here? This is helpful to track why we are adding this constraint and when we can remove it.

Reply:
added in https://github.com/pytorch/benchmark/pull/1829/commits/fce2df3619c366c5e574dd73682be76f9c80d5de

Reply:
Sorry, I mean to add link to the upstream GitHub issue, not the branch URL.

Reply:
ah, it's a typo.